packages/nx-plugin/src/generators/library/generator.spec.ts

```diff
@@ -31,7 +31,7 @@ describe('create-package generator', () => {
     const pkgJson = readJson(tree, paths.packageJson);
     expect(pkgJson.peerDependencies).toMatchInlineSnapshot(`
       {
-        "@fluentui/react-components": ">=9.25.1 <10.0.0",
+        "@fluentui/react-components": ">=9.35.1 <10.0.0",
```

Comment:
Does this matter, it was automatically updated by running test

Reply:
this is fine, all new packages creatd after this PR will have a higher lower bound for compatibility. It's overridable by each package after autogeneration in any case